qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -135,8 +152,20 @@ jobs:
         state: pending
     <% end %> # status_reporting
     on_failure:
+      <% if enable_terraform %>
+      try:
+        do:
+          - task: delete-aks-resource-group
+            file: ci/qa-pipelines/tasks/delete-aks-resource-group.yml
+            params:
+              << : *common-task-params
+          - put: pool.kube-hosts
+            params: {remove: pool.kube-hosts}
+      <% end %> # enable_terraform
```

Comment:
Use an else here instead

Reply:
I kept it this way : 

1.  we can just add additional condition in addition to !enable_terraform when required.
2. keep it consistent with linehttps://github.com/SUSE/cf-ci/blob/d78b41d1862cb40665707215fc3bf0d4cbc30145/qa-pipelines/qa-pipeline.yml.erb#L165
3. easy to read compared to else. At this point, I prefer readability in qa-pipeline.erb we desperately need it :)